.circleci/config.yml

```diff
@@ -318,6 +329,16 @@ workflows:
           <<: *filter-dev-branch
           requires:
             - deploy-staging
+  integration-tests-staging:
+    when:
+      equal: [integration-tests-staging, << pipeline.parameters.workflow_to_run >>]
```

Comment:
How do you change value of `pipeline.parameters.workflow_to_run` to  `integration-tests-staging` when it's a PR merge?

Reply:
PR merges should always trigger the build-deploy workflow (thus, it's the default value for the `workflow_to_run` parameter). The other workflows are only intended to be run manually.